aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -450,7 +453,7 @@ public boolean dropNamespace(Namespace namespace) throws NamespaceNotEmptyExcept
 
     glue.deleteDatabase(DeleteDatabaseRequest.builder()
         .catalogId(awsProperties.glueCatalogId())
-        .name(IcebergToGlueConverter.toDatabaseName(namespace))
+        .name(IcebergToGlueConverter.toDatabaseName(namespace, awsProperties.glueCatalogSkipNameValidation()))
```

Comment:
I think skipNameValidation should suffice (I don't think we need the glueCatalog part in the name)

Reply:
actually we have to, because we already have `glueCatalogSkipArchive()`...

Reply:
Ah good point, nvm @xingfanx !